src/example/pkg/example.go

```diff
@@ -0,0 +1,22 @@
+package pkg
+
+import (
+	"fmt"
+
+	"github.com/danielgtaylor/unistyle" // this is a third-party package
+)
+
+// Hello returns a greeting message
+func Hello(name string) string {
+	return fmt.Sprintf("Hello, %s!", name)
+}
+
+// HelloFancy returns a message using unixstyle - a third-party package
```

Comment:
Typo:
```suggestion
// HelloFancy returns a message using unistyle - a third-party package
```

Reply:
The package is called `unistyle`.  https://github.com/danielgtaylor/unistyle